lib/internal/Magento/Framework/MessageQueue/Topology/Config/ExchangeConfigItem/BindingInterface.php

```diff
@@ -7,6 +7,8 @@
 
 /**
  * Instances of this interface represent config binging items declared in etc/queue_topology.xsd
+ *
+ * @api
```

Comment:
Hi @mrtuvn,

Could you please share why this class is also marked as `@api`?

Reply:
https://github.com/magento/magento2/blob/1019b918b744fac5f461abb0bd87b77fdd27cfbd/lib/internal/Magento/Framework/Amqp/Topology/BindingInstallerInterface.php <= this class marked api. When you see this you can find this reference to Magento\Framework\MessageQueue\Topology\Config\ExchangeConfigItem\BindingInterface (non-api)

Reply:
You can see info interface listed in description